Wikipedia/UI-V5/WMFSelfSizingWaterfallCollectionViewLayout.m

```diff
@@ -0,0 +1,14 @@
+
+#import "WMFSelfSizingWaterfallCollectionViewLayout.h"
+
+@implementation WMFSelfSizingWaterfallCollectionViewLayout
+
+- (CGSize)collectionViewContentSize {
+    if ([self.collectionView numberOfSections] == 1 && [self.collectionView numberOfItemsInSection:0] == 0) {
```

Comment:
So, does this mean it won't bite us again under different circumstances? If not, we can merge this and get around to upstreaming later.  OTOH, if we think this could be a problem, let's go ahead and make it a development pod and fix the underlying issue.